python-libraries/data-platform-catalogue/data_platform_catalogue/entities.py

```diff
@@ -50,22 +66,41 @@ def from_data_product_metadata_dict(metadata: dict, version, owner_id: str):
             description=metadata["description"],
             version=version,
             owner=owner_id,
+            owner_display_name=metadata["dataProductOwnerDisplayName"],
+            maintainer=metadata.get("dataProductMaintainer", None),
+            maintainer_display_name=metadata.get(
+                "dataProductMaintainerDisplayName", None
+            ),
```

Comment:
None is the default returned if the key is not found. `metadata.get("dataProductMaintainerDisplayName" ),`